# uint32 type
string pixel_format
uint32 width
uint32 height
float32 fps
